brayns/Brayns.cpp

```diff
@@ -225,13 +233,7 @@ struct Brayns::Impl : public PluginAPI
             }
         }
 
-        if (_parametersManager.isAnyModified() || camera.isModified() ||
-            scene.isModified())
-        {
-            _engine->getFrameBuffer().clear();
-        }
-
-        _engine->getScene().getTransferFunction().resetModified();
+        scene.getTransferFunction().resetModified();
```

Comment:
hmm, this actually already done in commitTransferFunctionData(). we should remove it here then.

Reply:
OK